docroot/themes/humsci/humsci_basic/patterns/structured-card/structured-card.html.twig

```diff
@@ -0,0 +1,45 @@
+{%
+  set classes = [
+    'hb-card',
+    'hb-card--horizontal'
+  ]
+%}
+
+{%- set attributes = attributes.addClass(classes) -%}
+{%- set has_image = image|render|striptags('<drupal-render-placeholder><span><div><picture><source><img>')|trim -%}
+
+<div{{ attributes }}>
+  {% if has_image is not empty %}
+    <div class="hb-card__img">
+      {{ image }}
+    </div>
+  {% endif %}
+
+  <div class="hb-card__content">
+    {% if title %}
+      <div class="hb-card__title">{{ title }}</div>
+    {% endif %}
+
+    {% if subtitle %}
+      <div class="hb-card__subtitle">{{ subtitle }}</div>
+    {% endif %}
+
+    {% if description %}
+      <div class="hb-card__description">{{ description }}</div>
+    {% endif %}
+
+    {% if columns %}
+      <div class="hb-card__columns">{{ columns }}</div>
+    {% endif %}
+
+    {% if category %}
+      <div class="hb-card__category">
+        {{ category }}
+      </div>
+    {% endif %}
+
+    {% if bottom %}
+      <div class="hb-card__description">{{ bottom }}</div>
```

Comment:
I used the description class since it already has text styles and spacing built in. Ideally it would be nice to rename the class to be a bit more reusable but then we'd have to update all of the cards.